docs/3.api/3.utils/refresh-nuxt-data.md

```diff
@@ -61,9 +61,11 @@ async function refreshAll () {
 
 This example below refreshes only data where the key matches to `count`.
 
+::code-group
 ```vue [pages/some-page.vue]
 <script setup lang="ts">
 const refreshing = ref(false)
+const { data: count } = await useLazyAsyncData('count', () => $fetch('/api/count'))
```

Comment:
Perhaps we could consider adjusting the example to display a loading state when a refresh is in progress, and otherwise show the content.

The main intention would be to encourage users to make use of the refresh method provided by useAsyncData. What do you think?